docs/modules/secret-operator/pages/scope.adoc

```diff
@@ -12,34 +12,46 @@ The exact effect of the scope depends on which xref:secretclass.adoc#backend[`ba
 
 == Supported Scopes
 
-[#node]
-=== `node`
+[#listener-volume]
+=== `listener-volume`
 
-The `node` scope is resolved to the name of the Kubernetes Node object that the Pod is running on.
-This will typically be the DNS name of the node.
+*Syntax*: `listener-volume=<VOLUME>` (for example: `listener-volume=my-listener`)
+
+The `listener-volume` scope is resolved according to a xref:listener-operator:listener.adoc[Listener].
+
+The xref:listener-operator:listener.adoc[Listener] should be mounted as a xref:listener-operator:volume.adoc[volume] alongside the Secret volume, and takes the name of that
+volume as its parameter. This way, the scope will automatically adapt to the xref:listener-operator:listenerclass.adoc[ListenerClass] that it uses.
+
+Multiple xref:listener-operator:listener.adoc[Listener]s can be bound by specifying the scope multiple times.
+
+note:: The parameter is the name of the Pod's _volume_, not the name of the PersistentVolumeClaim, PersistentVolume, or xref:listener-operator:listener.adoc[Listener]. This allows the scope to bind to replica-specific or ephemeral xref:listener-operator:listener.adoc[Listener]s.
 
 [#pod]
 === `pod`
 
+*Syntax*: `node`
```

Comment:
typo?

Reply:
Goddamnit, copy/paste is dangerous
f17cea74efa9b9c120fec8e838588a2ada62336e